app/services/katello/event_queue.rb

```diff
@@ -0,0 +1,39 @@
+module Katello
+  class EventQueue
+    @event_types = {}
+
+    def self.clear_events(event_type, object_id, on_or_earlier_than)
+      Katello::Event.where(:in_progress => true, :object_id => object_id, :event_type => event_type).where('created_at <= ?', on_or_earlier_than).destroy_all
+    end
+
+    def self.next_event
+      first = Katello::Event.where(:in_progress => false).order(:created_at => 'asc').limit(1)
+      events = Katello::Event.where(:in_progress => false, :object_id => first.pluck(:object_id),
```

Comment:
On a dev environment just:

1) shut down your rails server (this stops dynflow)
2) in a rails console run:
```
 Katello::EventQueue.push_event("import_host_errata", 1)
```

where '1' is the host id.  Run this as many times as you want!
When you're ready to start pulling things off the queue you can just start your rails server.   You can also put invalid host ids on the queue to simulate an error.

A 2nd option would be to just comment out `::Actions::Katello::EventQueue::Monitor.ensure_running(world)
` in engine.rb and restart your rails server.  That'll keep the monitor from running.